projects/aca-shared/src/lib/services/app.extension.service.ts

```diff
@@ -492,7 +497,7 @@ export class AppExtensionService implements RuleContext {
     return false;
   }
 
-  runActionById(id: string, additionalPayload?: ModalConfiguration) {
+  runActionById(id: string, additionalPayload?: any) {
```

Comment:
Just to ensure - was that intentional change? Or should we add extra field to this configuration type (and if need - rename that configuration type)?

Reply:
Yes this was intentional as in the future we might want to pass various objects as additional payload so I'm not sure if we should define any type for that, we can but it can easily grow to one big interface having all params optional so that only one can be passed at the time, but I'm open for discussion about this one @AleksanderSklorz

Reply:
Yeah you are right, let's leave it @MichalKinas